packages/@react-spectrum/provider/chromatic/Provider.chromatic.tsx

```diff
@@ -38,6 +45,48 @@ storiesOf('Provider', module)
   .add(
     'custom theme',
     () => render({theme: THEME})
+  )
```

Comment:
Should this file be refactored to use the new syntax?

Reply:
at some point yes, if it's not hard you can do it here, if it takes more than a few min, maybe do it on its own later